### DISCUSSION TITLE
请问thuthesis.pdf本身是用latex写的吗？如果是的话很想要这个漂亮的模板。

### DISCUSSION THREAD

**Discussion by YangZ2020** [tuna/thuthesis#730] [Q&A] [answered]
Thanks！！！

**Comment by stone-zeng** [comment, marked as answer]
用的是 LaTeX 的 `ltxdoc` 文档类，然后用 `dtx-style` 宏包自定义了样式，后者也可以在 `thuthesis.dtx` 中找到：

https://github.com/tuna/thuthesis/blob/b49a994cf7d2929c7e70d04e2b78579953ba73f5/thuthesis.dtx#L6551-L6745
